R/timeseriessummary-data.R

```diff
@@ -119,10 +119,10 @@ constructTSDetails <- function(reportData, timezone){
     
     #Measurement Methods
     methodValue <- ""
-    methodStartTime <- NULL #Null so that it is hidden if there is no method
+    methodEndTime <- NULL #Null so that it is hidden if there is no method
     if(!isEmptyOrBlank(methodData) && !isEmptyVar(methodData)){
       methodValue <- methodData[1,][['methodCode']]
-      methodStartTime <- methodData[1,][['startTime']]
+      methodEndTime <- methodData[1,][['endTime']]
```

Comment:
I agree with your interpretation of the ticket that Method Start time should also be replaced on the report by end time but after thinking about it some more you might just want to confirm that with Laura (or perhaps she might even want both the start and end times listed?). Unless you already did that, in which case 👍